dll/cpl/intl/lang/ro-RO.rc

```diff
@@ -14,23 +14,23 @@ CAPTION "Opțiuni regionale"
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Standarde și formate", -1, 5, 5, 234, 162
-    LTEXT "Această opțiune afectează modul în care numerele, valutele, datele și orele sunt reprezentare în anumite aplicații.", -1, 14, 17, 220, 25
-    LTEXT "Selectați un element care să se potrivească cu proprietățile sale sau dați clic pe Personalizează pentru a vă alege propriile formate:", -1, 14, 37, 220, 22
+    LTEXT "Această opţiune se referă la modul în care unele programe formatează numerele, valorile monetare, datele şi ora.", -1, 14, 17, 220, 25
+    LTEXT "&Selectaţi un element pentru a-i prelua formatările sau faceţi clic pe Particularizare pentru a alege propriile formate:", -1, 14, 37, 220, 22
     COMBOBOX IDC_LANGUAGELIST, 14, 56, 150, 160, CBS_DROPDOWNLIST | CBS_NOINTEGRALHEIGHT | CBS_SORT | WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_VSCROLL
     PUSHBUTTON "&Particularizare…", IDC_SETUP_BUTTON, 168, 56, 62, 13
     LTEXT "Exemple", -1, 14, 73, 70, 10
     LTEXT "Număr:", -1, 16, 86, 68, 10
-    LTEXT "Valută:", -1, 16, 101, 68, 10
-    LTEXT "Timp:", -1, 16, 116, 68, 10
-    LTEXT "Dată (compact):", -1, 16, 131, 68, 10
-    LTEXT "Dată (extensiv):", -1, 16, 146, 68, 10
+    LTEXT "Simbol monetar:", -1, 16, 101, 68, 10
+    LTEXT "Oră:", -1, 16, 116, 68, 10
+    LTEXT "Dată scurtă:", -1, 16, 131, 68, 10
+    LTEXT "Dată lungă:", -1, 16, 146, 68, 10
     EDITTEXT IDC_NUMSAMPLE_EDIT, 89, 86, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
     EDITTEXT IDC_MONEYSAMPLE_EDIT, 89, 101, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
     EDITTEXT IDC_TIMESAMPLE_EDIT, 89, 116, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
     EDITTEXT IDC_SHORTTIMESAMPLE_EDIT, 89, 131, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
     EDITTEXT IDC_FULLTIMESAMPLE_EDIT, 89, 146, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
-    GROUPBOX "Locație geografică", -1, 5, 172, 234, 55
-    LTEXT "Specificați locația astfel încât unele ser&vicii să vă poată furniza informații locale; de exemplu, știri și rapoarte meteo.", -1, 14, 184, 210, 24
+    GROUPBOX "Amplasare", -1, 5, 172, 234, 55
+    LTEXT "P&entru ca serviciile să vă ofere informaţii regionale, cum ar fi ştirile sau vremea, selectaţi locul unde vă aflaţi:", -1, 14, 184, 210, 24
```

Comment:
Why did you turn the trailing `.` to a `:` ? That is not what our other languages do.

Reply:
I verified. You are right. I will revert this change.

Reply:
Fixed!